server.go

```diff
@@ -3250,11 +3250,11 @@ func (s *server) genNodeAnnouncement(features *lnwire.RawFeatureVector,
 	return *s.currentNodeAnn, nil
 }
 
-// updateAndBrodcastSelfNode generates a new node announcement
+// updateAndBroadcastSelfNode generates a new node announcement
 // applying the giving modifiers and updating the time stamp
-// to ensure it propagates through the network. Then it brodcasts
+// to ensure it propagates through the network. Then it broadcasts
 // it to the network.
-func (s *server) updateAndBrodcastSelfNode(features *lnwire.RawFeatureVector,
+func (s *server) updateAndBroadcastSelfNode(features *lnwire.RawFeatureVector,
```

Comment:
since we are touching the code, could you remove the `skip CI` and push it again? thanks!